### PR TITLE
Increase coverage

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: packages/controllers/src/snaps/SnapController.test.ts [hunk 1]
```diff
@@ -254,6 +255,35 @@ const getTruncatedSnap = ({
   } as const;
 };
 
+jest.mock('./utils', () => ({
+  ...(jest.requireActual('./utils') as any),
```
Comment (Mrtenz) [inline packages/controllers/src/snaps/SnapController.test.ts:259]: Doesn't this return `any` by default? Even better would be to specify the type, e.g. `jest.requireActual<typeof utils>('./utils')`.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: packages/controllers/src/snaps/SnapController.test.ts [hunk 2]
```diff
@@ -1416,6 +1448,53 @@ describe('SnapController', () => {
...
+
+      const result = await snapController.installSnaps('foo.com', {
+        [id]: {},
+      });
+
+      expect(result).toStrictEqual({
+        [id]: getTruncatedSnap({ initialPermissions }),
+      });
+      expect(fetchSnapMock).toHaveBeenCalled();
+      expect(callActionMock).toHaveBeenCalledTimes(3);
```
Comment (ritave) [inline packages/controllers/src/snaps/SnapController.test.ts:1496]: Might also want to check parameters provided to `callActionMock`

File: packages/controllers/src/snaps/SnapController.test.ts [hunk 3]
```diff
@@ -1849,4 +1928,106 @@ describe('SnapController', () => {
...
+          ),
+        )
+        .mockResponseOnce(sourceCode);
+
+      const id = 'local:https://localhost:8081';
+      const result = await controller.add({
+        origin: FAKE_ORIGIN,
+        id,
+      });
+      expect(fetchMock).toHaveBeenCalled();
```
Comment (ritave) [inline packages/controllers/src/snaps/SnapController.test.ts:1969]: You can use `toHaveBeenCalledTimes(2)` here

**Review by ritave — DISMISSED** [review]

File: packages/controllers/src/snaps/SnapController.test.ts (hunk 2 above)
Comment (FrederikBolding) [reply packages/controllers/src/snaps/SnapController.test.ts:1496]: done!

File: packages/controllers/src/snaps/SnapController.test.ts (hunk 3 above)
Comment (FrederikBolding) [reply packages/controllers/src/snaps/SnapController.test.ts]: done! it's actually called `3` times, but left a comment to explain that.

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**